### PR TITLE
Update README

### LINKED ISSUE #5: Update README
Currently it is empty. Let's write some short description what this tool does and how to use it

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

Fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # SymbiFlow tools package manager (STPM)
+
+This python package is used to retrieve the latest artifacts stored in Google Cloude Storage (GCS).
+The artifacts are uploaded by the SymbiFlow projects CI builds.
+
+## Usage
+
+This script has four non-required parameters to control which bucket and which artifact to get:
```
Comment (kgugala) [inline README.md]: optional?

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+- `--project`: name of the project/repository within the SymbiFlow organization
+- `--build_name`: name of the CI that has uploaded the desired artifact
+- `--jobset`: name of the jobset. This can assume two values: `presubmit`, `continuous`
+- `--get_max_int`: returns also the number of the latest build
```
Comment (kgugala) [inline README.md]: can you add the default values here?

**Comment by mithro** [comment]
I was wonder if this would be better called "symbiflow-tools-data-manager"? It doesn't really manage packages but the data required by the packages right?

**Comment by acomodi** [comment]
@mithro Agreed. I think also that `manager` might be incorrect as well, as this does not manage data, but it is rather a helper to get the data.

Maybe `symbiflow-tools-data-fetcher`?

**Comment by mithro** [comment]
I think `manager` in the name is good -- we should expand the tool to understand what has been downloaded. (IE You have all the artix line available, etc).

**Comment by acomodi** [comment]
@mithro Ok, I'll proceed with renaming the repo than (and the README as well)

**Comment by mithro** [comment]
BTW -- I'm thinking this tool should probably be included as part of the `symbiflow_XXXX` (synth/place/etc) tooling?

**Pull request merged**